src/main/java/com/diffblue/demo/ecommerce/models/Cart.java

```diff
@@ -6,8 +6,6 @@
 import java.util.HashMap;
 import java.util.Map;
 
-
-
```

Comment:
In a proper dev workflow these fixes should be squashed into the commit that created the problem in the first place (if this commit is in the same PR).